Resources/Prototypes/SS220/Entities/Weapons/Melee/ntbaton.yml

```
@@ -0,0 +1,48 @@
+# © SS220, An EULA/CLA with a hosting restriction, full text: https://raw.githubusercontent.com/SerbiaStrong-220/space-station-14/master/CLA.txt
+
+- type: entity
+  name: телескопическая бита НТ
+  parent: BaseItem
+  id: NTbaton
+  description: А вы знали, что офицеры ЦК любят играть в бейсбол головами некомпитентного руководства своих активов?
+  components:
+  - type: Sprite
+    sprite: SS220/Objects/Weapons/Melee/ntbaton.rsi
+    layers:
+    - state: telescope_off
+      map: [ "enum.ToggleVisuals.Layer" ]
+  - type: SwitchableWeapon
+    openSound: "/Audio/SS220/Weapons/telescopichka/batonextend.ogg"
+    closeSound: "/Audio/SS220/Weapons/telescopichka/batonextend.ogg"
+    damageOpen:
+      types:
+        Blunt: 28
+        Stamina: 120
+    damageFolded:
+      types:
+        Blunt: 0
+        Stamina: 0
+    sizeOpened: 15
```

```suggestion
    sizeOpened: 15
```
это и значение ниже скорее всего не будут верно работать, это работает теперь по гриду